README.md

```diff
@@ -35,12 +38,15 @@ https://www.npmjs.com/get-npm).
     import { Alert } from 'fundamental-react';
     ```
 
-
 ## Versioning
 
 The `fundamental-react` library follows [Semantic Versioning](https://semver.org/). These components strictly adhere to the [MAJOR].[MINOR].[PATCH] numbering system (also known as [BREAKING].[FEATURE].[FIX]).
 
-Merges to the `master` branch will be published as a prerelease.  Prereleases will include an **rc** version (_e.g._ [MAJOR].[MINOR].[PATCH]-rc.[RC]).
+Merges to the `master` branch will be published as a prerelease. Prereleases will include an **rc** version (_e.g._ [MAJOR].[MINOR].[PATCH]-rc.[RC]).
+
+## Known Issues
+
+There are no known major issues.
```

Comment:
I dont agree that we should put "There are no known major issues." here.

First, in the event this statement is false, it requires a version rev just to update the readme state as such.
Second, we are a pre-1.0 library. the version communicate that we are not set in terms of stablity.

I would rather this said:

```
## Issues
Please see https://github.com/SAP/fundamental-react/issues
```

Reply:
Agreed.  Maybe clean up the link a bit like:
```markdown
## Issues
Please see [Issues](https://github.com/SAP/fundamental-react/issues).
```

Reply:
updated